app/alerts.py

```diff
@@ -201,55 +194,54 @@ def build_vision_polygon(site_lat, site_lon, yaw, opening_angle, dist_km):
 # Fire alerts
 # The following block is dedicated to fetching information about fire alerts and displaying them on the map.
 
-def build_alerts_elements(img_url, alert_status, alert_metadata, map_style):
+def build_alerts_elements(images_url_live_alerts, live_alerts, map_style):
     """
     This function is used in the main.py file to create alerts-related elements such as the alert button (banner)
     or the alert markers on the map.
 
     It takes as arguments:
 
-    - 'img_url': the URL address of the alert frame to be displayed on the left of the map;
-    - 'alert_status': a binary variable indicating whether or not there is an ongoing alert to display;
-    - 'alert_metadata': a dictionary containing;
+    - 'images_url_live_alerts': url dict with live_alerts urls having event_id keys
+    - 'live_alerts': json containing live_alerts data
     - 'map_style': the type of map in place, either 'alerts' or 'risks'.
 
-    All these inputs are instantiated in the main.py file via a call to the API.
+    All these inputs are instantiated in the main.py file via a POST from the API.
 
     In the base case, the function returns:
 
-    - the URL address of the image to be displayed on the left of the map;
-    - the alert button (banner above the map);
-    - the alert markers displayed on the map.
+    - the urls addresses of the image to be displayed on the left of the map
+    - the new_alert_button
+    - the alert markers displayed on the map
+    - new navbar_color as in alert mode
+    - new navbar title
 
     But if the style of map in place is 'risks', we don't want to display neither the alert markers.
     So in this case, the third output of the function is a void string.
     """
 
-    # Fetching alert status and reusable metadata
-    alert_lat = alert_metadata["lat"]
-    alert_lon = alert_metadata["lon"]
-    alert_id = str(alert_metadata["id"])
-
-    if alert_status == 1:
-        # Building the button that allows users to zoom towards the alert marker
-        alert_button = dbc.Button(
-            children="Départ de feu, cliquez-ici !",
-            color="danger",
-            block=True,
-            id=f'alert_button_{map_style}'
-        )
-
-        # Format of the alert marker icon
-        icon = {
-            "iconUrl": '../assets/pyro_alert_icon.png',
-            "iconSize": [50, 50],        # Size of the icon
-            "iconAnchor": [25, 45],      # Point of the icon which will correspond to marker's and popup's location
-            "popupAnchor": [0, -20]      # Point from which the popup should open relative to the iconAnchor
-        }
-
-        # Building the list of alert markers to be displayed
-        alerts_markers = [dl.Marker(
-            id="alert_marker_{}".format(alert_id),   # Setting a unique id for each alert marker
+    # Changing the navabar color and title
```

Comment:
```suggestion
    # Changing the navbar color and title
```

Again a really dumb comment about a small typo